Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:18
+FROM node:18-slim
+RUN apt-get update && apt-get install -y openssl
```

Comment:
It always amuses me how long the shorter-named `apt` cli tool has been out and yet we all have `apt-get` deeply burnt into our muscle memory :heart:

Reply:
Indeed - good catch.

I even usually just use `apt` when typing in a terminal, but when it comes to scripts the `apt-get` raises its head again.

Will update